gradle.properties

```diff
@@ -94,3 +94,5 @@ enableMkdocs=false
 # set simpleSnapshot during build to prevent getProjectVersion() from making
 # a complex -SNAPSHOT version string with commit count & commit hash
 #simpleSnapshot=true
+woodstox=6.4.0
```

Comment:
Please change this to `woodstoxVersion` and add it just after `jdomVersion`

Reply:
Updated as suggested